lib/web/resources.go

```diff
@@ -174,6 +174,22 @@ func (h *Handler) createRoleHandle(w http.ResponseWriter, r *http.Request, param
 	return item, trace.Wrap(err)
 }
 
+func (h *Handler) getRole(w http.ResponseWriter, r *http.Request, params httprouter.Params, ctx *SessionContext) (interface{}, error) {
+	clt, err := ctx.GetClient()
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	roleName := params.ByName("name")
```

Comment:
Do we need to do any URL decoding here?

Reply:
I wish it was better documented, but no, we don't. :) In fact, I haven't see URL parameter escaping anywhere in our routers, and it seems that this is the very reason why we are using a fork of `julienschmidt/httprouter`. See https://github.com/gravitational/httprouter/pull/1/files.